reconcile/signalfx_endpoint_monitoring.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import sys
+from typing import Any, Optional
+
+from reconcile.closedbox_endpoint_monitoring_base import (
+    EndpointMonitoringProvider,
+    Endpoint,
+    parse_prober_url,
+    run_for_provider,
+)
+from reconcile.utils.openshift_resource import OpenshiftResource
+from reconcile.utils.semver_helper import make_semver
+
+QONTRACT_INTEGRATION = "signalfx-prometheus-endpoint-monitoring"
+QONTRACT_INTEGRATION_VERSION = make_semver(0, 1, 0)
+
+LOG = logging.getLogger(__name__)
+
+PROVIDER = "signalfx"
+
+
+def run(
+    dry_run: bool, thread_pool_size: int, internal: bool, use_jump_host: bool
+) -> None:
+    try:
+        run_for_provider(
+            PROVIDER,
+            build_probe,
+            QONTRACT_INTEGRATION,
+            QONTRACT_INTEGRATION_VERSION,
+            dry_run,
+            thread_pool_size,
+            internal,
+            use_jump_host,
```

Comment:
please consider the use of keyword arguments, especially for these long calls

Reply:
see [408dde8](https://github.com/app-sre/qontract-reconcile/pull/2268/commits/408dde8472ff90deacb8f955c98fe3c04b9c3eea)